cmd/package_test.go

```diff
@@ -186,3 +191,16 @@ func (s *suite) TestBuildPackage(c *C) {
 	c.Check(resultFile, Equals, filepath.Join(s.packageDir, "package-name.mpm"))
 	c.Check(resultFile, TarGzEquals, s.packageFiles)
 }
+
+func (s *suite) TestDescribePackage(c *C) {
+	// Prepare
+	ImportPackage(s.repo, s.packageDir)
+
+	// This is what we're testing here.
+	descr, err := DescribePackage(s.repo, "package-name")
+
+	// Expectations.
+	c.Assert(err, IsNil)
+	c.Check(descr, Not(Equals), "")
```

Comment:
Is this one necessary? With the next check you ensure that it contains a specific text.